routing/routing_integration_tests/routing_integration_tests.pro

```diff
@@ -13,6 +13,11 @@ TEMPLATE = app
 ROOT_DIR = ../..
 DEPENDENCIES = map routing search storage indexer platform geometry coding base osrm jansson protobuf tomcrypt succinct stats_client
 
+# this dependency is not built on Linux
```

Comment:
It is obvious, no? To explain why this dependency is not enabled on Linux. That is not obvious at all.